sdk/src/transaction_context.rs

```diff
@@ -893,6 +893,24 @@ impl<'a> BorrowedAccount<'a> {
         }
     }
 
+    /// Appends all elements in a slice to the account
+    #[cfg(not(target_os = "solana"))]
+    pub fn extend_from_slice(&mut self, data: &[u8]) -> Result<(), InstructionError> {
+        let new_len = self.get_data().len() + data.len();
+        self.can_data_be_resized(new_len)?;
+        self.can_data_be_changed()?;
+
+        if data.is_empty() {
```

Comment:
One thing to note is that even though the old lookup table didn't do this check before the account was `touch()?`ed, I think this is safe since early verify isn't enabled yet so none of these methods are failing yet.